### PR TITLE
Add module

### PR DESCRIPTION
**Pull request by deleon-jasminerobelle**
I added my module.

### CODE REVIEW

File: package/deleon.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            jasmine_info ()
+
+        elif choice == '2':
+            print("\n--- Goals ---")
+            print("1. Graduate with honors.")
+            print("2. Obtain a fulfilling IT career.")
+            print("3. Have a motorcycle.")
+
+        elif choice == '3':
+            print("\n--- Comment from Alejandro ---")
```
Comment (alejandro-aleckmcklaiyre) [inline package/deleon.py:32]: I will add my own comment here.

**Review by alejandro-aleckmcklaiyre — CHANGES_REQUESTED** [review]

File: package/deleon.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            print("\n--- Comment from Esparagoza ---")
+            print("---")
+
+        elif choice == '5':
+            print("\n--- Comment from Gomez ---")
+            print("---")
+
+        elif choice == '6':
+            print("\n--- Comment from Mosenos ---")
+            print("---")
```
Comment (ghost) [inline package/deleon.py]: I will add my comment here.

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

File: package/deleon.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            print("\n--- Comment from Alejandro ---")
+            print("That is a nice goal. Goodluck!")
+
+        elif choice == '4':
+            print("\n--- Comment from Esparagoza ---")
+            print("---")
+
+        elif choice == '5':
+            print("\n--- Comment from Gomez ---")
+            print("---")
```
Comment (gomez-ashleyhermione) [inline package/deleon.py]: I will add my comment here

**Review by gomez-ashleyhermione — CHANGES_REQUESTED** [review]

**Review by gomez-ashleyhermione — APPROVED** [review]

File: package/deleon.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            print("2. Obtain a fulfilling IT career.")
+            print("3. Have a motorcycle.")
+
+        elif choice == '3':
+            print("\n--- Comment from Alejandro ---")
+            print("That is a nice goal. Goodluck!")
+
+        elif choice == '4':
+            print("\n--- Comment from Esparagoza ---")
+            print("---")
```
Comment (esparagoza-mikkakette) [inline package/deleon.py:37]: ill add my comment here

**Review by esparagoza-mikkakette — CHANGES_REQUESTED** [review]

**Review by esparagoza-mikkakette — APPROVED** [review]

**Review by alejandro-aleckmcklaiyre — APPROVED** [review]

**Pull request merged**